src/components/Warnings/Warnings.jsx

```diff
@@ -1,12 +1,33 @@
 import classes from './Warnings.module.scss';
 import Warning from './Warning/Warning';
+import EmptyListMessage from '../EmptyListMessage/EmptyListMessage';
+import { useCookies } from 'react-cookie';
+import { useEffect, useState } from 'react';
+import ApiService from '../../services/ForecastApiService';
+import mockData from './mockData';
+
+const warningsList = warnings => warnings.map(warning => <Warning data={warning} />);
 
 function Warnings() {
+  const [warnings, setWarnings] = useState([]);
+
+  useEffect(async () => {
+    const { data } = await ApiService.getWarnings('105128581');
```

Comment:
'105128581' - ? again?

Reply:
I have removed this magic string and have connected redux to this component.